Src/Private/Get-AbrCPPMNetworkDevice.ps1

```diff
@@ -0,0 +1,131 @@
+function Get-AbrCPPMNetworkDevice {
+    <#
+    .SYNOPSIS
+        Used by As Built Report to returns Network Device settings.
+    .DESCRIPTION
+        Documents the configuration of Aruba ClearPass in Word/HTML/Text formats using PScribo.
+    .NOTES
+        Version:        0.1.0
+        Author:         Alexis La Goutte
+        Twitter:        @alagoutte
+        Github:         alagoutte
+        Credits:        Iain Brighton (@iainbrighton) - PScribo module
+
+    .LINK
+        https://github.com/AsBuiltReport/AsBuiltReport.Aruba.ClearPass
+    #>
+    [CmdletBinding()]
+    param (
+
+    )
+
+    begin {
+        Write-PScriboMessage "Discovering Network Device settings information from $System."
+    }
+
+    process {
+        PageBreak
```

Comment:
Sections should immediately follow the previous section. Is there a particular reason a pagebreak is required?

Reply:
it is only better display for me a new chapiter start of top of page but i can remove if needed

Reply:
I understand your reasoning however I'd also like reports to remain consistent in their outlay. I think sections should immediately follow the previous section.